### ISSUE TITLE
MVC model bound ID field not posted back

### ISSUE THREAD

**Issue by DiskJunky** [timdwilson/typeahead-mvc-model#8]
Once selecting an item from the returned results and clicking on a submit button, the ID value isn't posted back to the controller in the model

**Comment by timdwilson** [comment]
Please send me the source code/test where this is failing.  This works in every test I have run.  My suspicion is there is an underlying javascript error that is preventing the ID value from being set.  Please let me know, however, what the error is and/or the relevant test case so I can reproduce it.

**Comment by DiskJunky** [comment]
This works in your sample application. I suspect this may relate to issue #6 with the unusually long/complex autocomplete-id-field. Thankfully, I don't actually need this field in my app so I'm not too concerned but hopefully finding the cause of the complex id format will resolve this too.

**Comment by timdwilson** [comment]
Yes, that is correct.  I'll leave this issue open.  I think that this is related to trying to use properties on an item in a collection in AutocompleteFor.
